README.md

```diff
@@ -35,18 +35,21 @@ For blogs, release announcements and more, please checkout the [buildah.io](http
 
 ## Buildah and Podman relationship
 
-Buildah and Podman are two complementary Open-source projects that are available on
-most Linux platforms and both projects reside at [GitHub.com](https://github.com)
-with Buildah [here](https://github.com/containers/buildah) and
-Podman [here](https://github.com/containers/libpod).  Both Buildah and Podman are
-command line tools that work on OCI images and containers.  The two projects
-differentiate in their specialization.
-
-Buildah specializes in building OCI images.  Buildah's commands replicate all
-of the commands that are found in a Dockerfile. Buildah’s goal is also to
-provide a lower level coreutils interface to build images, allowing people to build
-containers without requiring a Dockerfile.  The intent with Buildah is to allow other
-scripting languages to build container images, without requiring a daemon.
+Buildah and Podman are two complementary open-source projects that are
+available on most Linux platforms and both projects reside at
+[GitHub.com](https://github.com) with Buildah
+[here](https://github.com/containers/buildah) and Podman
+[here](https://github.com/containers/libpod).  Both, Buildah and Podman are
+command line tools that work on Open Container Initiative (OCI) images images
+and containers.  The two projects differentiate in their specialization.
+
+Buildah specializes in building OCI images.  It allows building images with and
+without Dockerfiles while not requiring any root privileges.  Buildah’s
+ultimate goal is to provide a lower-level coreutils interface to build images.
+The flexibility of building images without Dockerfiles allows to integrate
```

Comment:
"allows to integrate" to "allows for the integration of"

Reply:
Thanks!